retry/retry.go

```diff
@@ -48,7 +52,7 @@ func (r Retry) Do(ctx context.Context, f func(context.Context) error, desc strin
 	for {
 		select {
 		case <-ctx.Done():
-			r.logger.Info("stopping retry", "description", desc)
+			r.logger.Info("stopping maxRetry", "description", desc)
```

Comment:
This is an info level log message, so it won't only be read be developers. `maxRetry` is a variable name that doesn't mean much for an 'operator'

Reply:
Was this incorrectly changed as part of a search and replace?

Reply:
it should be retry, this was a mistake Lorna identified as a bad automatic refactor by myself and goland.

This is fixed